partner/WHMCS/modules/servers/jetpack/jetpack.php

```diff
@@ -0,0 +1,234 @@
+<?php
+
+use WHMCS\Database\Capsule;
+
+
+/**
+ * A WHMCS module for use by Jetpack hosting partners to provision Jetpack plans.
+ * The module provides functionality for partner hosts to be able to save their
+ * client id and secret to request an access token for provisioning plans.
+ *
+ * Plans available for provisioning include free, personal, premium and professional
+ *
+ * A host has options to either provision(Create == WHMCS equivalent functional term)
+ * or Cancel(Terminate == WHMCS equivalent functional term) from the WHMCS client area.
+ *
+ * Host setup for custom fields is currently required in order to use the module.
+ *
+ */
+
+/**
+ * Jetpack Meta Data for WHMCS module.
```

Comment:
Is this phpdoc for the `WHMCS` check or for `jetpack_MetaData`?

Reply:
@nb This was for the jetpack_MetaData and not in the right spot after testing a few things. This has been updated. Thanks!